sites/public/__tests__/components/listing/ListingView.test.tsx

```diff
@@ -16,6 +16,7 @@ describe("<ListingView>", () => {
             applicationDueDate: pastDate,
           }}
           jurisdiction={jurisdiction}
+          googleMapsApiKey="GOOGLE_MAPS_KEY"
```

Comment:
do we really want this change?? passing in the API key like this as a prop is not something that I'm sure we'll need to do -- it doesn't need to be covered by React props because its state will never change, its current status as an env variable makes a lot more sense to me.